k8s/namespaces/jenkins/jenkins.yaml

```
@@ -22,7 +22,7 @@ spec:
       # Used for label app.kubernetes.io/component
       componentName: "jenkins-master"
       image: "hmctspublic.azurecr.io/jenkins/jenkins"
-      imageTag: "db533u"
+      imageTag: "db5k01"
```

build keeps failing -.-
```suggestion
      imageTag: "db5k05"
```